pkg/coreos/actuator_reconcile.go

```diff
@@ -189,3 +198,36 @@ func isContainerdEnabled(criConfig *extensionsv1alpha1.CRIConfig) bool {
 
 	return criConfig.Name == extensionsv1alpha1.CRINameContainerD
 }
+
+func enableCGroupsV2(cloudConfig *CloudConfig) ([]string, error) {
+	var additionalUnitNames []string
+
+	cloudConfig.CoreOS.Units = append(
+		cloudConfig.CoreOS.Units,
+		Unit{
+			Name:    "enable-cgroupsv2.service",
+			Command: "start",
+			Enable:  true,
+			Content: `[Unit]
+Description=Oneshot unit used to patch the kubeconfig for cgroupsv2.
```

Comment:
```suggestion
Description=Oneshot unit used to patch the kubelet config for cgroupsv2.
```

Reply:
If the patch is required or not, depends basically on the version of the Flatcar image.

If I see correctly, the chosen machineImage (=> version) is available in the controlplane webhook (as part of the Cluster definition). Thus it should be possible to implement it also there.